SphereSurfaceChargeDensity/PhysicsModule.py

```diff
@@ -0,0 +1,25 @@
+"""
+This Module is to define the main desired funtion:
+    A function to calculate the surface charge density of a conducting spherical shell held at constant potential.
+
+The specifications needed for the shell:
+    The radius (R): a float number.
+    The constant potential (V): a float number.
+
+Considering the units used, the default system will be the metric system.
+However, it will be flexible to convert to other systems by manually choosing the inpus' units.
+A constant value of \epsilon_0 is thus set by default to the units of the metric system.
+"""
+
+from MathOpsModule import *
+
+def sigma(R, V, epsilon_0 = 8.854*(10**(-12))):
```

Comment:
CONSTANTS should not be function arguments

Reply:
That was intended to be flexible with the choice of the systems of units. So, programming-wise, it's just a variable with a default value.

Reply:
You also probably do not want to change that during runtime which makes this a CONSTANT in any case